packages/inquirer/lib-cjs/inquirer-default.js

```diff
@@ -0,0 +1,3 @@
+const inquirer = require("./inquirer.js").default;
```

Comment:
This file sneaked in the diff (prob added/commited before the `.gitignore` change)

Reply:
Oh I see this is desired. Let's update the `.gitignore` so this file is included not ignored by Git.